core/controllers/payload_validator.py

```diff
@@ -27,15 +27,41 @@
 from typing import Any, Dict, List, Optional, Tuple, Union
 
 
+def get_schema_type(arg_schema: Dict[str, Any]) -> Any:
+    """This method returns the schema type for an argument by accepting the
+    schema for the corresponding argument.
```

Comment:
```suggestion
    """Returns the schema type for an argument by accepting the
    schema for the corresponding argument.
```

Also, I don't understand this sentence.

Reply:
Updated.